diff --git a/paperwork-gtk/src/paperwork_gtk/cmd/install.py b/paperwork-gtk/src/paperwork_gtk/cmd/install.py
index da34354c..4e060455 100644
--- a/paperwork-gtk/src/paperwork_gtk/cmd/install.py
+++ b/paperwork-gtk/src/paperwork_gtk/cmd/install.py
@@ -51,7 +51,7 @@ class Plugin(openpaperwork_core.PluginBase):
             datadir, 'applications', 'work.openpaper.Paperwork.desktop'
         )
         appdata_path = os.path.join(
-            datadir, "metainfo", "work.openpaper.Paperwork.appdata.xml"
+            datadir, "metainfo", "work.openpaper.Paperwork.metainfo.xml"
         )
 
         os.makedirs(os.path.dirname(desktop_path), exist_ok=True)
diff --git a/paperwork-gtk/src/paperwork_gtk/data/work.openpaper.Paperwork.appdata.xml b/paperwork-gtk/src/paperwork_gtk/data/work.openpaper.Paperwork.appdata.xml
index fea43913..32f1d95a 100644
--- a/paperwork-gtk/src/paperwork_gtk/data/work.openpaper.Paperwork.appdata.xml
+++ b/paperwork-gtk/src/paperwork_gtk/data/work.openpaper.Paperwork.appdata.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component xmlns="https://specifications.freedesktop.org/metainfo/1.0" type="desktop-application">
+<component type="desktop">
   <id>work.openpaper.Paperwork.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>Paperwork</name>
